res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1394,27 +1418,31 @@ PioneerDDJSX.tempoResetButton = function(channel, control, value, status, group)
 PioneerDDJSX.autoLoopButton = function(channel, control, value, status, group) {
     if (value) {
         if (engine.getValue(group, "loop_enabled")) {
-            engine.setValue(group, "reloop_exit", true);
+            engine.setValue(group, "reloop_toggle", true);
         } else {
-            engine.setValue(group, "beatloop_" + 4 + "_toggle", true);
+            engine.setValue(group, "beatloop_activate", true);
```

Comment:
beatloop_activate should be reset to 0 on button up or the button on screen will stay lit. reloop_toggle should also be reset to 0 (the button in skins lights up with loop_enabled, but just in case a skin doesn't do that, the controller script shouldn't leave reloop_toggle stuck at 1).